prin

```diff
@@ -53,7 +53,14 @@ PROG=${0##*/}
 # If NO ARGUMENTS should return *usage*, uncomment the following line:
 usage=${1:-yes}
 
-source $(dirname $(readlink -ne $BASH_SOURCE))/lib/common.sh
```

Comment:
I think this move would cause issues if this is PR'd after I clean `./prin`. I'll make this the next one I clean against `shellcheck`

Reply:
https://github.com/kubernetes/release/pull/763

This is my PR to clean prin

Reply:
@pswica In fact, I don't remove it, By the way I just added the "if" condition before it. Also I merge yours and fixed some `shellcheck` error, please review that.

Reply:
Great. This looks good from a shellcheck perspective. Thanks a lot for your patience while I clean our scripts against shellcheck

Reply:
Thanks again for your great efforts.